### DISCUSSION TITLE
1.8 RC1 release

### DISCUSSION THREAD

**Discussion by pubpub-zz** [pubpub-zz/ppInk#144] [Announcements]
new features
* enh : UI improvement : icons refreshed / Magnetic angle configurable / Window Mode border color configurable
* new : add new drawing in arrow heads allowing multiple customizable shapes
* new : add Spot mode to hide all but around cursor (configurable through options/general)
* new : add new filling to paint outside of shapes
* new : add tool to draw strokes of patterns
* fix : clear fickering artefacts during dynamic zoom on screen borders.
* fix : moving toolbar eased.
* enh : variable width strokes(drawn with tablet) are now saved/reloaded.
* enh : right click with clipart selected plots the image centered on the cursor
* new : add resize and rotate tools
* enh : add option (on pen tab) to allow to change line style through click (or hotkey) on an already selected pen
* enh : add option (on hotkey tab) to Enh to select which Linestyle are accessible through Linestyle Hotkey 
* fix : multiple fixes, see commits history for details

<hr /><em>This discussion was created from the release <a href='https://github.com/pubpub-zz/ppInk/releases/tag/v1.8RC1'>1.8 RC1 release</a>.</em>

**Comment by n1lotpal** [comment]
thank you mr. pp, you have created a great software. it could be even considered a premium software, as the extend of customisability is too much.

**Reply by pubpub-zz** [reply]
@n1lotpal, thanks for your compliment.Do not hesistate to star the project

**Comment by chuanfantasy** [comment]
how to use

**Reply by chuanfantasy** [reply]
I can't find the exe file

**Comment by pubpub-zz** [comment]
@yinchuanxihuanfantexi 
The build package is available here
https://github.com/pubpub-zz/ppInk/releases/download/v1.8RC1/ppInk64_1.8RC1.zip

**Comment by pubpub-zz** [comment]
@yinchuanxihuanfantexi, have you been able to download it ? do you enjoy it ?
